src/api/taxonomicService/GetTaxonomicService.ts

```diff
@@ -19,41 +18,32 @@ const GetTaxonomicService = async ({ handle }: { handle?: string }) => {
         const taxonomicServiceID: string = handle.replace(process.env.REACT_APP_HANDLE_URL as string, '');
 
         try {
-            let result: { data: JSONResult };
-            let mockData;
-
-            switch (taxonomicServiceID) {
-                case 'TEST/SEMKDJE98D7':
-                    mockData = AcceptedTaxonomicService;
-
-                    result = { data: { ...mockData } };
-
-                    break;
-                case 'TEST/LOFKDJE98D7':
-                    result = { data: { ...SuggestedTaxonomicService } };
-
-                    break;
-                case 'TEST/POTKDJE98D7':
-                    mockData = ReferenceCollection;
-
-                    result = { data: { ...mockData } };
-
-                    break;
-                default:
-                    throw (new Error('No mock data found'));
-            }
+            const result = await axios({
+                method: 'get',
+                url: '/Op.Retrieve',
```

Comment:
Is it a conscious decision to use the DOIP API endpoint? Cordra has regular API endpoints as well. I believe the DOIP API just calls the Rest API in the backend

Reply:
Ah ok, missed that :expressionless:

Reply:
Unfortunately what Sou commented is true. I even read in the docs they want to deprecate the normal HTTP API for outside calls